pkg/os/shell/shell_windows_test.go

```diff
@@ -8,6 +8,8 @@ import (
 )
 
 func TestDetect(t *testing.T) {
+	t.Skip("AppVeyor runs test with make/sh")
+
```

Comment:
Is there any way to detect that we are running in appveyor? Someone could be developping on Windows and running `make test` there, and they'd get messages about AppVeyor?